lib/asciidoctor/latex/node_processors.rb

```diff
@@ -194,7 +194,14 @@ def tex_process
       id ="_#{self.title.downcase.gsub(' ', '_')}"
       heading = "\\#{tagname}#{tagsuffix}\{#{self.title}\}"
       heading = $tex.hypertarget id, heading
-      value = "#{heading}\n#{self.content}"
+
+      if self.special && self.sectname == 'index'
```

Comment:
Hm, alright. I kinda copied that from https://github.com/asciidoctor/asciidoctor-pdf/blob/master/lib/asciidoctor-pdf/converter.rb#L426